src/types/domain/Commander.ts

```diff
@@ -3,8 +3,15 @@ export type Commander = {
     name: string;
     colorIdentity: string[];
     /**
-     * A collection of match IDs of matches that the commander is played in
+     * A collection of match IDs of matches that the commander is played in.
      */
     matches: string[];
+    /**
+     * Number of games we want to take into account when calculating stats (i.e. 4 player pods).
```

Comment:
nit: avoid using ie in this case. We don't know exactly how users will be using the player, but we can provide an example. (i believe IE implies a specific usage, but we don't want to imply it, just provide an example)

Something like

Number of matches we want to take into account when calculating stats. This number is determined after removing all games that don't match the minimum criteria to be considered for stats (for example: having the correct number of players). 

Long winded, but again, need to be crisp here

Reply:
fixed